src/flutter/shell/platform/linux_embedded/window/native_window_eglstream.cc

```diff
@@ -0,0 +1,435 @@
+// Copyright 2021 Sony Corporation. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "flutter/shell/platform/linux_embedded/window/native_window_eglstream.h"
+
+#include <fcntl.h>
+#include <sys/mman.h>
+#include <unistd.h>
+
+#include <unordered_map>
+
+#include "flutter/shell/platform/linux_embedded/logger.h"
+#include "flutter/shell/platform/linux_embedded/surface/cursor_data.h"
+
+namespace flutter {
+
+static constexpr char kCursorNameNone[] = "none";
+
+// Buffer size for cursor image. The size must be at least 64x64 due to the
+// restrictions of drmModeSetCursor API.
+static constexpr uint32_t kCursorBufferWidth = 64;
+static constexpr uint32_t kCursorBufferHeight = 64;
+
+NativeWindowEglstream::NativeWindowEglstream(const char* deviceFilename) {
+  drm_device_ = open(deviceFilename, O_RDWR | O_CLOEXEC);
+  if (drm_device_ == -1) {
+    LINUXES_LOG(ERROR) << "Couldn't open " << deviceFilename;
+    return;
+  }
+
+  ConfigureDisplay();
+
+  valid_ = true;
+}
+
+NativeWindowEglstream::~NativeWindowEglstream() {
+  if (drm_device_ == -1) {
+    return;
+  }
+
+  if (drm_crtc_) {
+    drmModeSetCrtc(drm_device_, drm_crtc_->crtc_id, drm_crtc_->buffer_id,
+                   drm_crtc_->x, drm_crtc_->y, &drm_connector_id_, 1,
+                   &drm_crtc_->mode);
+    drmModeFreeCrtc(drm_crtc_);
+  }
+
+  if (drm_property_blob_) {
+    drmModeDestroyPropertyBlob(drm_device_, drm_property_blob_);
+  }
+
+  if (drm_fb_) {
+    drmModeRmFB(drm_device_, drm_fb_);
+  }
+
+  close(drm_device_);
+}
+
+bool NativeWindowEglstream::Resize(const size_t width,
+                                   const size_t height) const {
+  if (!valid_) {
+    LINUXES_LOG(ERROR) << "Failed to resize the window.";
+    return false;
+  }
+
+  // todo: implement here.
+  LINUXES_LOG(ERROR) << "TODO: implement here!!";
+
+  return false;
+}
+
+bool NativeWindowEglstream::ShowCursor(double x, double y) {
+  // todo: implement here.
+  return true;
+}
```

Comment:
The ShowCursor function is not implemented, but the mouse cursor works fine (move, press, etc.).

Reply:
Will you create an new issue?

Reply:
I created https://github.com/sony/flutter-embedded-linux/issues/61.